components/connector-service/internal/certificates/certificates.go

```diff
@@ -43,7 +43,7 @@ func (cu *certificateUtility) decodeBytesFromBase64(bytes []byte) (decodedData [
 	data := make([]byte, base64.StdEncoding.DecodedLen(len(bytes)))
 	_, err := base64.StdEncoding.Decode(data, bytes)
 	if err != nil {
-		return nil, apperrors.Internal("Error while decoding base64 bytes: %s", err)
+		return nil, apperrors.Internal("Error while decoding base64 bytes. Internal error from base64 library: %s", err)
```

Comment:
'Internal error from base64 library' does not sound good, perhaps something like 'Invalid base64 string' would be better?

The same goes for latter occurrence in this file and the one in tests